src/long.ts

```diff
@@ -75,6 +75,21 @@ const INT_CACHE: { [key: number]: Long } = {};
 /** A cache of the Long representations of small unsigned integer values. */
 const UINT_CACHE: { [key: number]: Long } = {};
 
+const MAX_INT64_STRING_LENGTH = 22;
+
+/**
+ * @internal
+ * Validate an int64 string.
```

Comment:
If this code is kept as-it and doesn’t actually validate that the string is a 64-bit integer, only that it fits into that length in decimal and is an integer, then the doc comment should probably be clear about that (because I’d expect a method named `validateInt64String` to actually verify that the value is a valid 64-bit integer :))

Reply:
Offline discussion: I proposed we don't use a helper anymore so we can offer more granular error messages